### ISSUE TITLE
Composer install is throwing a seg fault on Ubuntu 14.04.

### ISSUE THREAD

**Issue by dsmithwfm** [composer/composer#5702]
Hello,

I'm building up a new Ubuntu 14.04 server and when I install composer I get a segfault.  I just built 2 servers (2 days ago) without issues.

```
$ curl -sS https://getcomposer.org/installer | php
All settings correct for using Composer
Segmentation fault (core dumped)
```

It created a 17Mb core file but, with gdb I wasn't able to track down anything obvious.  I'm not sure where to start looking next since our server is being built with the same scripts and composer hasn't been updated in the past few days.

**Comment by cmcfadden** [comment]
It's due to a bad /etc/ssl/certs/ca-certificates.crt.  Attached is one that works if you want to replace it and get up and running.  Not sure if ubuntu pushed a bad update or what.  (I just had an autoscale group start failing to build because of this)

Renamed .txt so github would take the attachment.

[ca-certificates.txt](https://github.com/composer/composer/files/488527/ca-certificates.txt)

**Comment by dsmithwfm** [comment]
Weird I did a diff of your crt file against mine and they are identical.  I've had this sort of thing happen months ago and the problem resolved itself.  I suspect you are right and something is happening on the Ubuntu install at his moment.

**Comment by Drewzar** [comment]
I just started getting this as well when doing `composer Install`'s. I have tried @cmcfadden 's updated ca-cert and that's not working as well.

**Comment by dsmithwfm** [comment]
Well I've found a workaround,  after doing a full "apt-get upgrade" it did fix Composer.  Hopefully this will resolve itself and we wont have to upgrade tons of packages.  This could introduce more issues.

```
# curl -sS https://getcomposer.org/installer | php
All settings correct for using Composer
Downloading 1.2.1...

Composer successfully installed to: /root/composer.phar
Use it: php composer.phar
```

**Comment by cmcfadden** [comment]
I realized I uploaded the wrong cert bundle earlier.  Sorry about that.  If you can't run apt-get upgrade for some reason, this should work?

I confirmed that an apt-get upgrade resolved the issue here as well.  Something definitely broke in ubuntu.  Weird.

[ca-certificates-proper.txt](https://github.com/composer/composer/files/488763/ca-certificates-proper.txt)

**Comment by dsmithwfm** [comment]
Thanks @cmcfadden the new cert file above ca-certificates-proper.txt fixed the install!

**Comment by dsmithwfm** [comment]
The proper fix for this in Ubuntu is to run the command to update ca certs.  Security wise this is safer than downloading a crt file from the internet and it fixed my issue.

```
sudo apt-get install --reinstall ca-certificates
```

**Comment by Dreyer** [comment]
Experienced this issue today in Ubuntu (Trusty) 14.04.2 LTS, seems to have resolved after upgrading to 14.04.5 using:  `apt-get update && apt-get upgrade` 

Reinstalling the CA certificates alone didn't resolve this issue in our case.

```
root@server:/# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.5 LTS
Release:	14.04
Codename:	trusty
```

**Comment by serhiikushch** [comment]
i faced the same issue on freebsd 10.2. can install composer only with --disable-tls flag
